### DISCUSSION TITLE
GitHub Desktop

### DISCUSSION THREAD

**Discussion by DirtyDragonSolly** [community/community#39150] [Other Feature Feedback, Questions, & Ideas] [answered]
Question

### Body

Is there a speed limit for uploading commits to the Internet in GitHub Desktop? If there is, how i can to remove it?

**Comment by steveward** [comment, marked as answer]
@DirtyDragonSolly GitHub Desktop doesn't place any limits on upload speed when pushing commits, but it will vary based on other factors like your bandwidth, latency, proximity to GitHub's servers, and how large the push operation itself is. I hope that helps!

**Reply by DirtyDragonSolly** [reply]
Thank you. It really helped.
